package.json

```diff
@@ -6,14 +6,14 @@
   "main": "d3/query-graphs.js",
   "scripts": {
     "test": "mocha --watch --recursive",
-    "bundle": "browserify d3/query-graphs.js -d -o d3/query-graphs.bundle.js",
-    "bundle-list": "browserify d3/query-graphs.js --list",
+    "bundle": "browserify d3/app.js -d -o d3/query-graphs.bundle.js",
+    "bundle-list": "browserify d3/app.js --list",
     "prebundle-uglify": "npm run bundle",
     "bundle-uglify": "uglifyjs --compress --mangle -- d3/query-graphs.bundle.js > d3/query-graphs.min.js",
     "prebundle-tlv": "npm run bundle-uglify",
     "bundle-tlv": "tar -czf $npm_package_name-tlv-qt-$npm_package_version.tgz -C d3 query-graphs.tlv.html query-graphs.css query-graphs.min.js",
     "postinstall": "npm run bundle-tlv",
-    "lint": "eslint upload-server.js d3/query-graphs.js d3/common.js d3/colors.js d3/hyper.js d3/tableau.js d3/json.js d3/xml.js d3/tql.js"
+    "lint": "eslint upload-server.js d3/app.js d3/query-graphs.js d3/common.js d3/colors.js d3/hyper.js d3/tableau.js d3/json.js d3/xml.js d3/tql.js"
```

Comment:
missing rename to tree-rendering.js?

Reply:
fixed.
Note how in the followup commit, this line is changed to `eslint .`, thereby including all files by default. The `.eslintignore` makes sure we don't lint the bundled and minified source